src/mixins/filing-template-mixin.ts

```diff
@@ -435,10 +435,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     })
 
     // restore NAICS
-    this.setRegistrationNaics(draftFiling.registration.business.naics || EmptyNaics)
+    this.setRegistrationNaics(draftFiling.registration.business?.naics || EmptyNaics)
 
     // restore Business Number
-    this.setRegistrationBusinessNumber(draftFiling.registration.business.taxId || null)
+    this.setRegistrationBusinessNumber(draftFiling.registration.business?.taxId || null)
```

Comment:
For both of these fixes... in some cases, the initial draft registration looks like this:

![image](https://user-images.githubusercontent.com/10002889/184944041-2400f81b-9d5b-4fb5-b6e7-9e0326cf4307.png)

As you can see, there is no `registration.business` object.